packages/views/file_upload.jade

```diff
@@ -0,0 +1,12 @@
+template(name="fileUpload")
+  .input-group
+    .input-group-btn
+      label.btn.btn-default.btn-file
+        span Choose file
+        i.fa.fa-folder-open
+        input.upload(name=name type="file")
+    .file-name-wrap
+      input.form-control.file-name(value="{{fileName}}" class="{{hasFile}}" readonly)
```

Comment:
Should I change the template's name? I suppose it's only getting the file, not uploading it. Is that what you mean?
Will add a test.